hazelcast/src/main/java/com/hazelcast/nio/IOUtil.java

```diff
@@ -275,17 +275,17 @@ public static Object readAttributeValue(ObjectDataInput in) throws IOException {
     }
 
     /**
-     * Closes the Closable quietly. So no exception will be thrown. Can also safely be called with a null value.
-     *
-     * @param closeable the Closeable to close.
+     * Quietly attempts to close a {@link Closeable} resource, swallowing any exception.
+     * @param closeable the resource to close. If {@code null}, no action is taken.
      */
     public static void closeResource(Closeable closeable) {
-        if (closeable != null) {
-            try {
-                closeable.close();
-            } catch (IOException e) {
-                Logger.getLogger(IOUtil.class).finest("closeResource failed", e);
-            }
+        if (closeable == null) {
```

Comment:
I'm fine with this change, since you go from one working form of code to another; but what is the reasoning behind it?